cmd/server/assets/codes/issue.html

```diff
@@ -123,8 +123,26 @@ <h1>{{t $.locale "codes.issue.header"}}</h1>
         <div class="card mb-3 shadow-sm">
           <div class="card-header">{{t $.locale "codes.issue.sms-text-message-header"}}</div>
           <div class="card-body">
+            {{if $currentRealm.SMSTextAlternateTemplates}}
             <div class="row form-group">
-              <label for="symptomDate" class="col-sm-6 col-md-4 col-lg-3">{{t $.locale "codes.issue.sms-text-message-label"}}</label>
+              <label for="sms-template" class="col-sm-6 col-md-4 col-lg-3">{{t $.locale "codes.issue.sms-template-label"}}</label>
+              <div class="col-sm-6 col-md-8 col-lg-9">
+                <div class="input-group">
+                  <select class="form-control" name="sms-template">
+                    <option value="Default SMS template">Default SMS template</option>
+                    {{range $k, $v := $currentRealm.SMSTextAlternateTemplates}}
+                    <option value="{{$k}}">{{$k}}</option>
```

Comment:
Should this be `{{$v}}`? If not, why is `$v` scoped?

Reply:
templates can't do `$k, _` and single var is just values. I need just the keys (labels)

Reply:
but then can do `$k := range`

Reply:
but `$k := range .someMap` iterates the map values, not the keys. Maybe I'm misunderstanding

Reply:
My b - apparently this is backwards in Go templates from the Go language /tableflip